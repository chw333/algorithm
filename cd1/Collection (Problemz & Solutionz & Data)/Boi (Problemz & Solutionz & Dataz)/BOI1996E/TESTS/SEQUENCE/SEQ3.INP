7
34
55
22
71
40
60
13