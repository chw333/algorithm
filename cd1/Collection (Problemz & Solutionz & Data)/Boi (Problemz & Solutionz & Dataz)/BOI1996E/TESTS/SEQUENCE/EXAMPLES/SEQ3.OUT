1
77 1
22 1
22 1
11 1